tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapRaceTest.java

```diff
@@ -94,47 +74,37 @@ public void verify() throws Exception {
         assertEquals("There should not be any data races", 0, failures);
     }
 
-    @Performance
-    public long getOperationCount() {
-        return operations.get();
+    @RunWithWorker
+    public AbstractWorkerTask createWorker() {
+        return new Worker();
     }
 
-    private class Worker implements Runnable {
-        private final Random random = new Random();
+    private class Worker extends AbstractMonotonicWorkerTask {
```

Comment:
Why is this called Task? It is already a worker. I would call it MonotonicWorker; but if you want to keep abstract in it call it AbstractMonitonicWorker.